source/mongodb-handshake/handshake.md

```diff
@@ -407,8 +407,19 @@ class DriverInfoOptions {
 
 Note that how these options are provided to a driver is left up to the implementer.
 
-If provided, these options MUST NOT replace the values used for metadata generation. The provided options MUST be
-appended to their respective fields, and be delimited by a `|` character. For example, when
+### Metadata updates after MongoClient initialization
+
+Drivers MUST provide an API that allows appending `DriverInfoOptions` to a `MongoClient` instance after initialization.
```

Comment:
We usually give an example API (or at least function signature) that implementors can reference. Could we add that here?

For example, see how startSession is documented: https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.md#mongoclient-changes